src/compiler/scala/tools/nsc/typechecker/Typers.scala

```diff
@@ -327,8 +332,9 @@ trait Typers extends Adaptations with Tags with TypersTracking with PatternTyper
         debuglog(s"Inferring view from $from to $to for $tree (reportAmbiguous= $reportAmbiguous, saveErrors=$saveErrors)")
 
         val fromNoAnnot = from.withoutAnnotations
-        val result = inferImplicitView(fromNoAnnot, to, tree, context, reportAmbiguous, saveErrors) match {
-          case fail if fail.isFailure => inferImplicitView(byNameType(fromNoAnnot), to, tree, context, reportAmbiguous, saveErrors)
+        val toNorm = normalizeProtoForView(to)
+        val result = inferImplicitView(fromNoAnnot, toNorm, tree, context, reportAmbiguous, saveErrors) match {
```

Comment:
Is there a sensible / fast path through `inferImplicitView` when `pt = WildCardType`?

Reply:
Sorry, I don't follow. If the overloaded signatures don't agree, `proto.underlying` will be `WildcardType`, so that will be `normalizeProtoForView`'s result, but a wildcard for the expected type of a view makes little sense. Are you saying we should add another check to avoid calling `inferImplicitView` for such a `pt`? I suggest that would be a separate issue, since this PR doesn't really add new ways we could get here with a wildcard expected type (the path from `adapt`/`isCoercible` would not get to their callsite of `inferImplicitView` with such a pt.